scripts/build-images

```diff
@@ -21,7 +21,7 @@ xargs -n1 -t docker image pull --quiet << EOF >> build/images-core.txt
     ${REGISTRY}/rancher/klipper-helm:v0.6.4-build20210813
     ${REGISTRY}/rancher/pause:${PAUSE_VERSION}
     ${REGISTRY}/rancher/mirrored-jettech-kube-webhook-certgen:v1.5.1
```

Comment:
Need to update this to the one used by the new chart.

You might be able to hold off on this PR though, as I'm about to merge https://github.com/rancher/rke2/pull/1827 which has these updates rolled in already.